### ISSUE TITLE
[Power Up] Feeding many trolls

### ISSUE THREAD

**Issue by Praveen91299** [XanaduAI/QHack2021#32]
### Team Name: 

Qumulus Nimbus

### Project Description: 

We provide a pennylane implementation of single qubit universal quantum classifier similar to that presented in [1] and [2]. We then provide an efficient method to parallely process classical data using a qram setup for the universal single qubit classifier.  
We then attempt to address quantum classifiers by data reuploading for **Quantum Data** for experiments when we have copies of the quantum state and show it's performance, which we believe has not been done before.  

We use the universal quantum classifier method and measurement strategies described in [1] to demonstrate a method of quantum music learning and generation by recasting the classifier into a markov chain like setup. We also use the qram structure we developed to combine and generate music.  
[1] - https://quantum-journal.org/papers/q-2020-02-06-226/  
[2] - https://github.com/AlbaCL/qhack21f
  
### Source code: 

[Repository](https://github.com/Praveen91299/qhack-qumulus-nimbus)  (incomplete, check the jupyter notebook here)

### Resource Estimate: 

We will be using Rigetti Aspen 9 to test out our quantum data quantum classifier, because of it's connectivity and the fact that our method that we propose deals with a qubit only once allowing us to move around the main qubit from one place to another.  

we will be using IonQ's fully connected qubit system to implement our music prediction system for a qram of upto 3 qubits which would require 6 fully connected qubits with high fidelity, thus using it.  

We would be using up to 800 to 1000 dollars worth, especially our music prediction requires multiple training and time samples, and any remaining would be useful later on.

**Comment by glassnotes** [comment]
@Praveen91299 thanks for the submission!

**Comment by co9olguy** [comment]
Thanks for your Power Up Submission @Praveen91299 !

To help us keep track of final submissions, we will be closing all of the [Power Up] issues. We ask you to open a new issue for your final submission. Please use [this pre-formatted [Entry] Issue template](https://github.com/XanaduAI/QHack/issues/new?assignees=&labels=submission&template=open-hackathon-final-submission.md&title=%5BENTRY%5D+Your+Project+Title). Note that for the final submission, the Resource Estimate requirement is replaced by a Presentation item.
